src/routes/inngang/inngang-fullmakt.tsx

```diff
@@ -0,0 +1,133 @@
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import Lenkepanel from 'nav-frontend-lenkepanel';
+import { FnrInput, Label } from 'nav-frontend-skjema';
+import React, { useState } from 'react';
+import styled from 'styled-components/macro';
+import { useBreadcrumbs } from '../../breadcrumbs/use-breadcrumbs';
+import { InngangKategori, Kategori } from '../../kategorier/kategorier';
+import { usePageInit } from '../../page-init/page-init';
+import { MarginTopContainer } from '../../styled-components/common';
+import { ContentContainer } from '../../styled-components/content-container';
+import { InngangMainContainer } from '../../styled-components/main-container';
+import { CenteredPageTitle } from '../../styled-components/page-title';
+import { SectionTitle } from '../../styled-components/section-title';
+import { WhiteSection } from '../../styled-components/white-section';
+import { User } from '../../user/user';
+import { getFullName } from '../klageskjema/summary/personlige-opplysninger-summary';
+import { foedselsnrFormat } from '../klageskjema/summary/text-formatting';
+import queryString from 'query-string';
+import { hasFullmaktFor } from '../../api/api';
+
+interface Props {
+    inngangkategori: InngangKategori;
+    kategori: Kategori;
+}
+
+const FieldWithButton = styled.div`
+    display: flex;
+    flex-flow: row wrap;
+    justify-items: space-around;
+    > * {
+        flex-basis: 180px;
+        margin: 10px 0;
+    }
+    justify-content: flex-start;
+`;
+
+const InngangFullmakt = ({ inngangkategori, kategori }: Props) => {
+    const { title, temaKey } = kategori;
+    usePageInit(`${title} \u2013 klage på vegne av andre`);
+    useBreadcrumbs([], 'Klage på vegne av andre');
+
+    const [fodselsnummer, setFodselsnummer] = useState<string | null>('');
```

Comment:
Hm, denne vil aldri bli satt til `null` av `FnrInput` og har heller ikke `null` som initial value.
Virker som om type `string` er mer korrekt.

Da må `handleSubmit` justeres litt igjen.
@mahval

Reply:
Jeg lagde en commit for dette og litt annen smårydding, @mahval.